integration/helpers.go

```diff
@@ -534,6 +534,33 @@ func (r *rpcTest) getBalance(address common.Address) (*hexutil.Big, error) {
 	return &u, nil
 }
 
+func (r *rpcTest) estimateGas(
+	from common.Address,
+	data []byte,
+	gas uint64,
```

Comment:
what is gas here? gas limit or gas price? better to rename

Reply:
Good point :+1: 
Updated in https://github.com/onflow/flow-evm-gateway/pull/81/commits/fac85cb67c8431ed48f45c170d5ecfbd9c1fb2cf